scrapenews/spiders/mg.py

```diff
@@ -49,29 +50,28 @@ class MGSpider(SitemapSpider):
     def parse(self, response):
         canonical_url = response.xpath('//link[@rel="canonical"]/@href').extract_first()
 
-        ## Skip excluded sections
+        # Skip excluded sections
         section = response.css('a.section').xpath('text()').extract_first()
         if section.lower() in IGNORE_SECTIONS:
             self.logger.info("Skipping %s because section is %s", canonical_url, section)
             return
 
-        ## Skip syndicated content
+        # Skip syndicated content
         body_html = "".join(response.css("#body_content p").extract())
         body_text = remove_tags(body_html, encoding='utf-8')
         for string in SKIP_STRINGS:
             suffix = body_text[-20:]
-            if unicode(string, 'utf-8') in suffix:
-                self.logger.info("Skipping %s because suffix %r contains %r",
-                                 canonical_url,
-                                 suffix,
-                                 string)
+            if six.text_type(string) in suffix:
```

Comment:
Not super important, but is `six´ even needed if we're just gonna be working with python 3 (unicode) strings from now on?